src/Query/Query.ts

```diff
@@ -139,38 +139,42 @@ export class Query implements IQuery {
     private parseHideOptions({ line }: { line: string }): void {
         const hideOptionsMatch = line.match(this.hideOptionsRegexp);
         if (hideOptionsMatch !== null) {
-            const option = hideOptionsMatch[1].trim().toLowerCase();
+            const hide = hideOptionsMatch[1] === 'hide';
```

Comment:
I had to think quite hard about the logic on line 142 and 174. I didn't spot the `!` on 174 the first time I read the code.

I could certainly live with this code remaining as it is, if you wanted to put this PR down.

But I think that the code would be easier to understand and maintain if the new option was called `hideUrgency` and it started with a default value of true.

That way, the naming and logic is the same for all the options, which will reduce the chances of bugs creeping in, in future.

Reply:
Yes, I think you're right. The small benefit of having the default value to be always false is not worth it. Changed it now.